documentation/SupportingDocuments/API_Documentation/Traffic_Influence_API_Documentation.md

```diff
@@ -120,7 +119,7 @@ Creates a new TrafficInfluence resource
 
 ##### Description
 
-Takes as input an object containing the intents from the API Consumer and creates a TrafficInfluence resource accordingly. The trafficInfluenceID parameter, that is part of the object, must not be valorized when creating a new resource. For this reason the trafficInfluenceID parameter must be avoided in the object, anyway it will be ignored by the API Producer. It is automatically generated by the system and returned in the response.
+Gets in input an object containing the intents from the API Consumer and creates a TrafficInfluence resourse accordingly. The trafficInfluenceID parameter, that is part of the object, must not be valorized when creating a new resource. For this reason the trafficInfluenceID parameter must be avoided in the object, anyway it will be ignored by the API Producer. It is automatically generated by the system and returned in the response.
```

Comment:
"Takes as input" is correct (not "Gets in input")

Reply:
Right, I will fix it

Reply:
this was wrong also in the YAML file, I'm going to fix it also there